utils/aoa-proxy/Makefile

```diff
@@ -0,0 +1,66 @@
+#
+# Copyright (C) 2023 Jó Ágila Bitsch
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+ 
+PKG_NAME:=aoa-proxy
+PKG_VERSION:=0.2
+PKG_RELEASE:=1
+ 
+PKG_BUILD_DIR:=$(BUILD_DIR)/aoa-proxy-$(PKG_VERSION)
+PKG_SOURCE:=aoa-proxy-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://github.com/jo-bitsch/$(PKG_NAME)/archive/refs/tags/v$(PKG_VERSION).tar.gz?
+PKG_HASH:=39f53a4724f542e87ec0aa7b05ea62be
+
+PKG_MAINTAINER:=Jó Ágila Bitsch <jgilab@gmail.com>
+PKG_LICENSE:=BSD-3-Clause
+PKG_LICENSE_FILES:=LICENSE
+ 
+include $(INCLUDE_DIR)/package.mk
+
+TARGET_CFLAGS += -DNO_HID=1
+ 
+define Package/aoa-proxy
+  SECTION:=utils
+  CATEGORY:=Network
+  TITLE:=Interact with Android devices using the Android Open Accessory Protocol
+  URL:=https://github.com/jo-bitsch/aoa-proxy
+  DEPENDS:=+libusb-1.0
+  PKG_BUILD_DEPENDS:=+argp-standalone
+endef
+ 
+define Package/aoa-proxy/description
+  This program aims to make working with Android Open Accessory (in 
+  particular v1) as easy as possible.
+  It has 2 modes, depending on the state of the attached Android device.
+  (1) Announce mode: When the Android device is not yet in AOA mode, this
+      program announces its identity to the Android device and tries to
+      put it into AOA mode.
+  (2) Forwarding mode: Forward all input from stdin to the AOA device and
+      all output from AOA to stdout.
+  For OpenWrt, it includes a hotplug script and a service to automatically
+  announce itself and forward AOA traffic to SSH/dropbear.
+endef
+ 
+define Build/Configure
+  $(call Build/Configure/Default)
```

Comment:
Why are you calling defaults again?

Reply:
My bad. I removed the Build/Configure definition.